### ISSUE TITLE
Actions are in More menu while there's plenty of space.

### ISSUE THREAD

**Issue by OlafvdSpek** [phpmyadmin/phpmyadmin#13949]
![pma](https://user-images.githubusercontent.com/2218936/35269397-21d8f680-002c-11e8-9f5a-775678153bb8.png)

**Comment by mauriciofauth** [comment]
This is a side effect of #13905.

**Comment by OlafvdSpek** [comment]
OK, could it be fixed?

**Comment by mauriciofauth** [comment]
Yes. But it must be implemented in a different way, because the previous implementation was unusable when the table had too many columns.

**Comment by OlafvdSpek** [comment]
If it's just icons I don't think any collapsing is needed.

**Comment by mauriciofauth** [comment]
For now you can disable the [`$cfg['HideStructureActions']`](https://docs.phpmyadmin.net/en/latest/config.html#cfg_HideStructureActions).

`$cfg['HideStructureActions'] = false;`

**Comment by OlafvdSpek** [comment]
Much better, thanks!

I think that should be the default for just icons.
